tokenize-scripts/00_createReferenceScript.sh

```diff
@@ -41,16 +42,16 @@ echo -e "\nCreating Minting Reference:\n" ${mint_script_reference_utxo}
 echo -e "\033[0;36m Gathering UTxO Information  \033[0m"
 ${cli} query utxo \
     --testnet-magic ${testnet_magic} \
-    --address ${reference_address} \
-    --out-file tmp/reference_utxo.json
+    --address ${seller_address} \
```

Comment:
The reference address will ONLY be for holding the scripts on chain. The transaction will be paid by NEWM (the seller address)